### DISCUSSION TITLE
Am I violating any license by distributing the documentation ?

### DISCUSSION THREAD

**Discussion by algovoid** [slint-ui/slint#6458] [Q&A] [answered]
Hi,

I have this [repo](https://github.com/algovoid/slint-offline-docs) that have the latest documentation (offline format) extracted from the www-releases using github workflows and , i need these offline docs for 2 reasons :

1. I have a slow, unreliable internet connections, and it is not guaranteed to have access to the online docs.

2. I am developing a VS Code extension that will offer the capability to view the online docs in a side panel inside VS Code as demonstrated in the image below, and I have a setting in the extension that will allow the user to choose between the online view or use an offline version (which will be downloaded using the aforementioned repository).

![image](https://github.com/user-attachments/assets/38fa8788-5ce6-4018-b4af-a50daa0fa968)

So my questions are : 
- Am I allowed to keep the repository public and have it provide the latest stable docs ?
- If not, then can I make the repo private and keep using it ?

Thanks.

**Comment by ogoffart** [comment, marked as answer]
I think you're fine.

The license of the documentation is the same as the license of Slint. And I think the docs mention that. So you should be allowed to publish it.

Regarding the documentation on a side panel of VSCode, It would even make sense eventually to have that included in our extension.

**Reply by algovoid** [reply]
Thank you for answering.

**Reply by hunger** [reply]
You should be able to find license info for any file that does not have a copyright header in `.reuse/dep5`.

This is the relevant section AFAICT:

```
Files: api/cpp/docs/*/*.html api/cpp/docs/*/*.css docs/*/*.css docs/*/*.html docs/*/book.toml docs/*/package.json
Copyright: Copyright © SixtyFPS GmbH <info@slint.dev>
License: MIT
```

It's all MIT license AFAICT, have fun :-)

I would greatly appreciate if you could consider to just contribute that sideline docs viewer into the official slint extension. That would be super awesome.
